2007-12-04  Ulrich Drepper  <drepper@redhat.com>

	* locales/si_LK (mon): Fix February and November spelling.
	Patch by Danishka Navin <snavin@redhat.com>.

--- libc/localedata/locales/si_LK	30 Jul 2006 22:19:43 -0000	1.2
+++ libc/localedata/locales/si_LK	5 Dec 2007 06:52:01 -0000	1.3
@@ -114,7 +114,7 @@ abmon       "<U0DA2><U0DB1>";/
 %
 % Full month names (%B)
 mon         "<U0DA2><U0DB1><U0DC0><U0DCF><U0DBB><U0DD2>";/
-            "<U0DB4><U0DD9><U0DB6><U0DBB><U0DC0><U0DCF><U0DBD><U0DD2>";/
+	    "<U0DB4><U0DD9><U0DB6><U0DBB><U0DC0><U0DCF><U0DBB><U0DD2>";/
             "<U0DB8><U0DCF><U0DBB><U0DCA><U0DAD><U0DD4>";/
             "<U0D85><U0DB4><U0DCA><U200D><U0DBB><U0DD2><U0DBA><U0DD9><U0DBD><U0DCA>";/
             "<U0DB8><U0DD0><U0DBA><U0DD2>";/
@@ -123,7 +123,7 @@ mon         "<U0DA2><U0DB1><U0DC0><U0DCF
             "<U0D85><U0D9C><U0DDD><U0DC3><U0DCA><U0DAD><U0DD4>";/
             "<U0DC3><U0DD0><U0DB4><U0DCA><U0DAD><U0DD0><U0DB8><U0DCA><U0DB6><U0DBB><U0DCA>";/
             "<U0D94><U0D9A><U0DCA><U0DAD><U0DDD><U0DB6><U0DBB><U0DCA>";/
-            "<U0DB1><U0DD9><U0DC0><U0DD0><U0DB8><U0DCA><U0DB6><U0DBB><U0DCA>";/
+            "<U0DB1><U0DDC><U0DC0><U0DD0><U0DB8><U0DCA><U0DB6><U0DBB><U0DCA>";/
             "<U0DAF><U0DD9><U0DC3><U0DD0><U0DB8><U0DCA><U0DB6><U0DBB><U0DCA>"
 %
 % Equivalent of AM PM
